agent/README.md

```diff
@@ -21,7 +21,6 @@ Set development env variables to .env file:
 ```
 KONTENA_URI=ws://kontena.local:4040
 KONTENA_TOKEN=secret_grid_token
-AGENT_NAME=vagrant_agent_1
```

Comment:
Not to be confused with the `KONTENA_NODE_NAME`... looking through the git history, the `AGENT_NAME` (or `KONTENA_AGENT_NAME` refers to the name of the `kontena-agent` container, but these are not used anymore.